test/formulas/reference/testcase.js

```diff
@@ -94,6 +93,27 @@ module.exports = {
         'SUM(INDEX(D1:E2, 0, 0, 1))': 20,
     },
 
+    OFFSET: {
+        'OFFSET(A1, 4,4)': 6,
+        'OFFSET(B7, 3,3)' : 6,
+        'OFFSET(B7, -1,1)' : 40,
+        'OFFSET(B7, 3,-1)' : 'Walnuts',
+        'OFFSET(B7, -1,-1)': 'Pears',
+        'OFFSET(B7, 0,0)': 2.8,
+        'OFFSET(B7, -7, 0)': FormulaError.REF,
+        'OFFSET(B7, 0, -3)': FormulaError.REF,
+        'SUM(OFFSET(B7, 1, 1, 2, 2))': 46,
```

Comment:
@marhong2 Let's use CONCAT instead of sum to combine arrays.

Reply:
Currently testcase does not allow arrays to be on the right hand side of the ":" more specifically it will compare the first value in the right hand array to the entire returned array. Thus concatting it would prove difficult to compare with a correct answer. If you wish I can work on fixing that bug/effect as a separate Lido task. TLDR we need to summarize the output of the left hand side as a single value

Reply:
Bah, okay, thanks for the explaination